HappyFox Test Automation Task Java /HelpDeskProductTests/src/main/java/Admin_Portal/AdminPortalTest1.java

```diff
@@ -0,0 +1,35 @@
+package Admin_Portal;
+
+import org.seleniumhq.jetty9.util.log.LoggerLog;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import CommonTests.CommonTests1; 
+
+ public class AdminPortalTest1 extends CommonTests1  {
+
+  @Test
+  public void Scenario1() {
```

Comment:
Scenario1 is not understand at least command should be added what scenario is going to automate